.idea/codeStyles/Project.xml

```diff
@@ -3,18 +3,9 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="java.util" alias="false" withSubpackages="false" />
```

Comment:
oops, these files were changed always by AndroidStudio after I've set it as in Evgenii Jemerov's video.

I've always stashed them, but now I've committed accidentally. Should I revert them?